.github/workflows/kind-workflow.yaml

```diff
@@ -0,0 +1,106 @@
+name: kind-workflow
+on:
+  pull_request:
+    branches:
+      - fission-github-actions
+    types: [labeled]
+jobs:
+  fission-on-kind:
+    runs-on: ubuntu-latest
+    if: github.event.label.name == 'kind'
+    steps:
+      - name: Free up some space
+        run: |
```

Comment:
Just trying to understand why these two commands are needed?

Reply:
This was added because sometime GitHub runner's storage gets full and it gives `no space left on device error`. GitHub runner are configured with limited resources as mentioned in the official document. 
https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners#supported-runners-and-hardware-resources

Issue raised on GitHub:
https://github.com/actions/virtual-environments/issues/709